4.x/typescript/Recenter/tsconfig.json

```diff
@@ -2,18 +2,17 @@
   "compilerOptions": {
     "module": "amd",
     "noImplicitAny": true,
-    "lib": ["ES2019", "DOM"],
     "esModuleInterop": true,
-    "sourceMap": true,
+    "lib":["DOM","ES2020"],
     "jsx": "react",
     "jsxFactory": "tsx",
     "target": "ES5",
+    "allowSyntheticDefaultImports": true,    
```

Comment:
I think this is already true when esModuleInterop is true.

Reply:
Confirmed and updated.